### ISSUE TITLE
When the component loses focus, the combo box list should close

### ISSUE THREAD

**Issue by greatislander** [greatislander/pressbooks-multiselect#143]
## Describe the bug

Currently when focus is shifted out of the combo box list using the keyboard, the list is left open. This can cause [2.4.11 Focus not obscured](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html) violations.

### Expected behavior
When focus leaves the combo box list, the list should be dismissed/collapse.

### Related issues

https://github.com/pressbooks/private/issues/1617
